sdk/messaging/azure-messaging-eventhubs/vcpkg/vcpkg.json

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Why does the diffing tool show this as red background?  Is there something special about this copyright?

Reply:
nope

Reply:
github is looking at the # and saying "This is not valid JSON, you might want to fix it".

For <reasons>, the checked in vcpkg.json file has a copyright notice, we strip it when we push the changes to vcpkg.